docs/references/architecture/readme.md

```diff
@@ -108,7 +108,7 @@ The Tauri-Apps organisation maintains two "upstream" crates from Tauri, namely T
 
 ### [TAO](https://github.com/tauri-apps/tao)
 
-Cross-platform application window creation library in Rust that supports all major platforms like Windows, macOS, Linux, iOS and Android. Written in Rust, it is a fork of [winit](https://github.com/rust-windowing/winit) that we have extended for our own needs like menu bar and system tray.
```

Comment:
Can we keep this bit of information?

Reply:
Well, the purpose of that sentence is to explain the purpose of cloning and extending `winit`. 
The addition of `like menu bar and system tray` is information that reinforces the main point of the sentence, which was already established.
However, as I previously mentioned in a comment, this emphasis could be left in if so desired.

Reply:
Yeah, I think the extra details here is needed.